__tests__/endpoints.test.js

```diff
@@ -110,13 +110,64 @@ describe('GET /api/articles', () => {
           expect(article).toHaveProperty('article_img_url');
           expect(article).toHaveProperty('comment_count')
           expect(article).not.toHaveProperty('body')
-        });                
+        })
+      });
+  });
+});
+        
+describe('GET /api/articles/:article_id/comments', () => {
+  test("Status 200: Should respond with comments for an article filtered by article_id order by latest comments", () => {
+    return request(app)
+      .get('/api/articles/6/comments')
+      .expect(200)
+      .then(({ body }) => {
+        expect(Array.isArray(body)).toBe(true);
+        expect(body).toBeSorted({ descending: true, key: 'created_at' });
```

Comment:
I'd consider checking for the length before your forEach to prevent and false positives if the array was empty.